server/chat-plugins/datasearch.js

```diff
@@ -419,6 +422,29 @@ function runDexsearch(target, cmd, canAll, message) {
 				continue;
 			}
 
+			if (target.endsWith(' asc') || target.endsWith(' desc')) {
+				if (parameters.length > 1) return {reply: `The parameter '${target.split(' ')[1]}' cannot have alternative parameters`};
+				let stat = target.split(' ')[0];
+				switch (toId(stat)) {
```

Comment:
This seems copied from the zboosts code. This should probably be broken out somewhat else.

Reply:
Like, a separate function outside?